src/index.css

```diff
@@ -57,7 +68,7 @@ body {
 .card-title {
 display: inline;
 text-align: left;
-font-size: 24px;
+font-size: 20px;
```

Comment:
The `.artist-name` also has a `font-size` of 20px on mobile

Reply:
thank you, adjusted. I shall be more careful

Reply:
No worries
This comes with time
Then you know more and more what to look out for